Makefile

```diff
@@ -187,7 +190,7 @@ docker-%: system-checks
 all: binaries
 android-dist: genconfig android
 
-$(RESOURCES_DOT_GO): $(NPM)
```

Comment:
If NPM was not around or was not in the PATH this target failed without any useful output.